### PR TITLE
Updated libraries

### PR DESCRIPTION
**Pull request by rid-hrant**
- Updated gradle plugin version to 3.1.3
- Updated compileSdkVersion, buildToolsVersion, targetSdkVersion to 27
- Updated support lib version to 27.1.1
- Changed deprecated dependency api to new

### CODE REVIEW

File: dexter/build.gradle [hunk 1]
```diff
@@ -2,12 +2,12 @@ apply plugin: 'com.android.library'
...
 android {
-  compileSdkVersion 25
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 27
+  buildToolsVersion '27.0.3'
 
   defaultConfig {
-    minSdkVersion 10
-    targetSdkVersion 25
+    minSdkVersion 14
```
Comment (pedrovgs) [inline dexter/build.gradle]: Hi @rid-hrant I'm taking a look at this PR and I was wondering why you've increased the ``minSdkVersion``. This change is huge right now. As we can review in these pages, the rest of the updates are correct and we are using the latest version released:

https://developer.android.com/studio/releases/gradle-plugin
https://developer.android.com/studio/releases/build-tools
https://developer.android.com/topic/libraries/support-library/revisions

But we should review this minSdkVersion carefully right @Serchinastico? I'm ok if we update the min SDK but we should keep it in mind and at least add this information to the release notes.

**Comment by pedrovgs** [comment]
I forgot to thank you @rid-hrant xD Thanks you so much for your contribution :smiley:

**Comment by pedrovgs** [comment]
I forgot to thank you @rid-hrant xD Thanks you so much for your contribution :smiley:

File: dexter/build.gradle (hunk 1 above)
Comment (rid-hrant) [reply dexter/build.gradle]: Hi my friend :smile: . I try to use dexter in my updated gradle version poroject, but it has conflict with support libs, so I decided to contribute and update libs and gradle version.
Min sdk version 10 has conflict with new support libs 27 version, so I had to update version to 14, 14 is android stidio's recomendid version.

Thanks for answering.
Sorry for bad english

**Comment by pedrovgs** [comment]
Ups I didn't read your comment @rid-hrant 

> Min sdk version 10 has conflict with new support libs 27 version, so I had to update version to 14, 14 is android stidio's recomendid version.

I'm ok updating the min SD version. Let's see what the rest of the team says about this.

**Comment by rid-hrant** [comment]
OK @pedrovgs I'll waiting reaction of the team.

**Comment by Serchinastico** [comment]
I'll be reviewing it next thursday :+1:

**Comment by rid-hrant** [comment]
@Serchinastico thank you to! :smile:

**Pull request merged**